adapter_pipelines/ss2_single_sample/adapter_example_static.json

```diff
@@ -0,0 +1,23 @@
+{
+  "AdapterSmartSeq2SingleCell.gtf_file": "gs://broad-dsde-mint-dev-teststorage/reference/GRCh38_Gencode/gencode.v27.primary_assembly.annotation.gtf",
+  "AdapterSmartSeq2SingleCell.hisat2_ref_trans_name": "gencode_v27_trans_rsem",
+  "AdapterSmartSeq2SingleCell.rrna_intervals": "gs://broad-dsde-mint-dev-teststorage/reference/gencode.v27.rRNA.interval_list",
+  "AdapterSmartSeq2SingleCell.star_ref_index": "gs://broad-dsde-mint-dev-teststorage/reference/GRCh38_Gencode/GRCh38_GencodeV27_Primary.tar",
+  "AdapterSmartSeq2SingleCell.hisat2_ref_index": "gs://broad-dsde-mint-dev-teststorage/reference/HISAT2/genome_snp_tran.tar.gz",
+  "AdapterSmartSeq2SingleCell.genome_ref_fasta": "gs://broad-dsde-mint-dev-teststorage/reference/GRCh38_Gencode/GRCh38.primary_assembly.genome.fa",
+  "AdapterSmartSeq2SingleCell.hisat2_ref_trans_index": "gs://broad-dsde-mint-dev-teststorage/reference/HISAT2/gencode_v27_trans_rsem.tar.gz",
+  "AdapterSmartSeq2SingleCell.rsem_ref_index": "gs://broad-dsde-mint-dev-teststorage/reference/rsem_ref/gencode_v27_primary.tar",
+  "AdapterSmartSeq2SingleCell.gene_ref_flat": "gs://broad-dsde-mint-dev-teststorage/reference/GRCh38_gencode.v27.refFlat.txt",
+  "AdapterSmartSeq2SingleCell.hisat2_ref_name": "genome_snp_tran",
+  "AdapterSmartSeq2SingleCell.stranded":"NONE",
+  "AdapterSmartSeq2SingleCell.reference_bundle": "bf51d668-3e14-4843-9bc7-5d676fdf0e01",
+  "AdapterSmartSeq2SingleCell.format_map": "gs://broad-dsde-mint-dev-teststorage/format_map_example.json",
```

Comment:
@ambrosejcarr I remembered @jishuxu has made a new format map file on the dev bucket (the format_map_example_SS2_latest.json`)for the new ss2, so that those files are now shown as unknown on the Ingest side, would you mind update it here? Or more easily, we rename that new file to this name (format_map_example.json).

Reply:
I have updated the file names in both staging and dev bucket.

Reply:
Hi Rex, confirming that based on your actions, we don't need to do any further modifications and this is good to merge?

Reply:
@ambrosejcarr No, I renamed the filename in the bucket to match this line, we don't need to change anything here!